operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -2102,6 +2138,10 @@ results-cache:
     annotations: {}
     labels: {}
 
+  appProtocol:
+    # -- Set the optional service protocol. Ex: "tcp"
+    client: null
```

Comment:
Adding this as a separate comment because it's a different topic: can you help me understand why we _wouldn't_ want to set the appProtocol by default? I tried setting it to `grpc` in a cluster without istio and nothing broke 😄 but that's not a very rigorous test

Reply:
Well actually I took this approach from the tempo charts, both the values structure and the choice of not adding it be default. I guess this is the most flexible way of doing it, if a deployment has multiple ports you can just extend the structure and when you run a different mesh requiring a different app-protocol you can easily configure it.